duradmin/src/main/webapp/WEB-INF/jsp/spaces/spaces-manager.jsp

```diff
@@ -16,9 +16,9 @@
                             type: '${fn:toLowerCase(storeOption.storageProviderType)}',
                             writable: ${storeOption.writable}
 							},
-							</c:forEach>				
+							</c:forEach>
```

Comment:
This formatting change probably should be reverted or at least added as a separate formatting themed commit since it is not related to duracloud-1214 (as far as I can tell).